stable_baselines3/common/envs/multi_input_envs.py

```diff
@@ -0,0 +1,266 @@
+from typing import Dict, Tuple, Union
+
+import gym
+import numpy as np
+
+from stable_baselines3.common.type_aliases import GymStepReturn
+
+
+class SimpleMultiObsEnv(gym.Env):
+    """
+    Base class for GridWorld-based MultiObs Environments 4x4  grid world.
+
+    .. code-block:: text
+
+        ____________
+       | 0  1  2   3|
+       | 4|¯5¯¯6¯| 7|
+       | 8|_9_10_|11|
+       |12 13  14 15|
+       ¯¯¯¯¯¯¯¯¯¯¯¯¯¯
+
+    start is 0
+    states 5, 6, 9, and 10 are blocked
+    goal is 15
+    actions are = [left, down, right, up]
+
+    simple linear state env of 15 states but encoded with a vector and an image observation
```

Comment:
so the image is just a copy of the vector?
wouldn't be better to have complementary information in order to check that the implementation is actually working?

Reply:
The image is not a copy of the vector as they are both independently randomly generated.
For every row there is an image, and for every column there is a vector. Only by combining them can an agent infer what state they are in.

Reply:
took me some time to understand it ^^" now it makes sense and I've added comments too